troubadix/troubadix.py

```diff
@@ -97,10 +97,17 @@ def generate_patterns(
 def from_file(include_file: Path, term: Terminal) -> Iterable[Path]:
     """Parse the given file containing a list of files into"""
     try:
-        return [
-            Path(f)
-            for f in include_file.read_text(encoding="utf-8").splitlines()
-        ]
+        # Return file list and remove duplicate files
+        return list(
+            set(
+                [
+                    Path(f)
+                    for f in include_file.read_text(
+                        encoding="utf-8"
+                    ).splitlines()
+                ]
+            )
+        )
```

Comment:
Just return the set here. A set is also an Iterable

Reply:
I get some errors with it.
  File "/home/work/work/troubadix/troubadix/troubadix.py", line 162, in main
    first_file = files[0].resolve()
TypeError: 'set' object is not subscriptable

Reply:
Ok in that case the typing of the return value of the `from_file` function should be changed to `List` too

Reply:
Ah sorry missed it.